membership_account_balance/README.rst

```diff
@@ -0,0 +1,27 @@
+Membership Management - Member Account Balance
```

Comment:
Point of Sales
Eric CAUDAL --
*Eric Caudal* /[Founder and CEO]/
Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 
8017/27/37 (Office)
*Elico Shanghai* (Hong Kong/Shenzhen/Singapore) 
<http://www.elico-corp.com> /*Odoo Gold Partner, best Odoo Partner 2014 
for APAC*/
On 11/23/2015 03:20 PM, Rona wrote:
>
> In membership_account_balance/README.rst 
> <https://github.com/Elico-Corp/odoo/pull/45#discussion_r45576554>:
>
> > @@ -0,0 +1,27 @@
> > +Membership Management - Member Account Balance
>
> @elicoidal <https://github.com/elicoidal> Do you know which 
> project_repo does these module push to? I shoud add the project_repo 
> in the README.rst.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Elico-Corp/odoo/pull/45/files#r45576554>.
>